core/src/connection/pool/task.rs

```diff
@@ -39,11 +39,16 @@ use futures::{
     SinkExt, StreamExt,
 };
 use std::pin::Pin;
-use void::Void;
 
-/// Commands that can be sent to a task.
+/// Commands that can be sent to a task driving a pending connection.
 #[derive(Debug)]
-pub enum Command<T> {
+pub enum PendingConnectionCommand {
+    Abort,
+}
+
+/// Commands that can be sent to a task driving an established connection.
+#[derive(Debug)]
+pub enum EstablishedConnectionCommand<T> {
```

Comment:
If my assumption above is correct, I'd vote for undoing this rename to reduce the noise of the diff.